Plugins/Public/npc_control/Main.cpp

```diff
@@ -503,8 +509,16 @@ void CreateNPC(wstring name, Vector pos, Matrix rot, uint iSystem)
 		// below shows the use of multiple part names.
 		FmtStr pilot_name(0, 0);
 		pilot_name.begin_mad_lib(16163); // ids of "%s0 %s1"
-		pilot_name.append_string(rand_name());  // ids that replaces %s0
-		pilot_name.append_string(rand_name()); // ids that replaces %s1
+		if (arch.Infocard != 0) {
```

Comment:
hm, can we rely on uint in a struct being initialised to 0? this feels like potentially UB

Reply:
Hi @dsyalex, I have added initialization of the ints just before they get set. Hope this is okay.